src/app/core/hq-dashboard-sub-menu/hq-dashboard-sub-menu.component.spec.ts

```diff
@@ -5,6 +5,9 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { environment } from '../../../environments/environment';
 import { RouterModule } from '@angular/router';
 import { BASE_PATH } from '../../variables';
+import { HasRoleDirective } from 'src/app/_directives/hasRole.directive';
```

Comment:
If we are using the _ convention for folder names could you move the two directives I created (autosize and preventdefault) into this folder?

Reply:
Sure, will do.